src/app/app.component.ts

```diff
@@ -11,7 +14,65 @@ import { DEPARTURE_AIRCRAFT_STATUS } from 'src/assets/mock-data/mock-aircraft-st
 })
 export class AppComponent {
 
-  ngOnInit() { }
+  private aircraftStatusUrl = "/assets/mock-data/mockAircraftStatus.json";
+  private tripsUrl = "assets/mock-data/mockTrips.json";
+  isDataAvailable = false;
+
+  departureAircraftStatus: AircraftStatus | undefined;
+  currentBoardingTime: any;
+  estimatedBoardingTime: any;
+  boardingTimeString: any;
+  boardingTimeDifference: any;
+  boardingTimeEarlyOrLate: any;
+  boardingTimeMinOrMins: any;
+  boardingLastRefresh: any;
+  boardingGate: any;
+
+  trip: any;
+
+  departureMonth: any;
+  departureDateTimeString: any;
+  arrivalMonth: any;
+  arrivalDateTimeString: any;
+
+  constructor(private http: HttpClient) { }
+
+  ngOnInit() {
+    this.getJSON(this.tripsUrl).pipe(
+      switchMap((data) => {
+        return forkJoin({
+          trips: of(data),
+          aircraftStatus: this.getJSON(this.aircraftStatusUrl)
+        });
+      })
```

Comment:
This can be done like this
```ts
   combineLatest([this.getJSON(this.tripsUrl), this.getJSON(this.aircraftStatusUrl)]).subscribe(([trips, airStatus]) => {
   /** Do what you want here */
   })
```